### PR TITLE
Updated user as/branch

### PR DESCRIPTION
**Pull request by focused220**
The user mutation will work as normal. The update to the homeBeach data occurs prior to returning. The query for the front will look like this:
`mutation{
  addUser( 
    cognitoUserId:"52",
    fullName:"Rob's TestUser132", 
    email:"testuser132@email.com", 
    homeBeach: 0, 
    homeBeachName:"", 
    longitude:-117.596621, 
    latitude: 35.621683){
    fullName,
    email,
    homeBeach,
    homeBeachName,
    longitude,
    latitude
  }
}`

and should return:
`{
  "data": {
    "addUser": {
      "fullName": "Rob's TestUser132",
      "email": "testuser132@email.com",
      "homeBeach": "778",
      "homeBeachName": "Arroyo De La Cruz Beach",
      "longitude": -117.596621,
      "latitude": 35.621683
    }
  }
}`

### CODE REVIEW

File: api/homeBeach.js [hunk 1]
```diff
@@ -3,82 +3,106 @@ const MongoClient = require('mongodb').MongoClient;
...
+    
+                //Update to user object
+                newUser.homeBeach = homeBeach[0].ID;
+                newUser.homeBeachName = homeBeach[0].NameMobileWeb;
+                
+                
+                
+                 
+                
+                //Update to user entry in DB - Not currently needed in this iteration
```
Comment (jthicks91) [inline api/homeBeach.js]: I like this for whats to come, looks good!

File: schema/schema.js [hunk 2]
```diff
@@ -24,7 +26,8 @@ const {
   GraphQLInt,
   GraphQLList,
   GraphQLNonNull,
-  GraphQLFloat
+  GraphQLFloat,
+  GraphQLInterfaceType
```
Comment (jthicks91) [inline schema/schema.js]: I would like to see how this was used, only cause im curious.

File: api/homeBeach.js [hunk 3]
```diff
@@ -3,82 +3,106 @@ const MongoClient = require('mongodb').MongoClient;
...
+                const db = client.db("howsthewater");  
+                
+                //Searches DB for locations within defined latitude and longitude constraints and
+                //puts each location into an array
+                const distances = [];
+                const allBeaches = [];
+                const beaches = await db.collection('locations').find({$and: [{LATITUDE: {$lt: userLocale.latUpper}}, 
+                    {LATITUDE: {$gt: userLocale.latLower}}]}).toArray().then(beaches =>{
+        
+                    //Converts degrees to miles and pushes beach ID and mileage to new array
```
Comment (jthicks91) [inline api/homeBeach.js]: this is really cool!

File: schema/schema.js (hunk 2 above)
Comment (focused220) [reply schema/schema.js]: I didn't end up having to use the interface after all. I should probably remove that :+1:

**Pull request merged**